### PR TITLE
The production version of the agb attribute issued by Nuts.

### PR DESCRIPTION
**Pull request by woutslakhorst**

### CODE REVIEW

**Comment by woutslakhorst** [comment]
@sietseringers

**Comment by confiks** [comment]
Please note that the issuer keypair has only six bases. That means that you cannot issue credentials with more than six attributes. For now that's probably fine, but it's something to consider now when creating the issuer (it's the `-a` option of `irmatool genkeypair`). 

A few additional suggestions:

- Looking at how the information is presented at https://www.agbcode.nl, I think "AGB-code", capitalized and with a hyphen, would be the best.
- Do you really think it's necessary to include "Vektis" in the credential name. Isn't "AGB-code" already identifying enough? Vektis can then be named in the credential description.
- The credential description is not really telling very much. Maybe the acronym could be explained. There's a small blurb on https://www.agbcode.nl that might be suitable.
- "unieke identificerende sleutel" should be "uniek identificerende sleutel"

File: nuts/Issues/agb/description.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	<Description>
+		<en>Vektis agb attributes issued by Nuts.</en>
+		<nl>Vektis agb attributen uitgegeven door Nuts.</nl>
+	</Description>
+
+	<Attributes>
+		<Attribute id="agbcode">
+			<Name>
+				<en>agbcode</en>
+				<nl>agbcode</nl>
```
[the remark this answers is not in the document]
Comment (woutslakhorst) [reply nuts/Issues/agb/description.xml]: Changed to AGB-code

File: nuts/Issues/agb/description.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Vektis agb by Nuts</en>
+		<nl>Vektis agb via Nuts</nl>
```
[the remark this answers is not in the document]
Comment (woutslakhorst) [reply nuts/Issues/agb/description.xml]: Changed to AGB

**Pull request merged**